### PR TITLE
[WIP] RHDEVDOCS-6146: Update Condition management commands section to follow correct style

### PR DESCRIPTION
**Pull request by Maharshi-Project**
Version(s): `pipelines-docs-1.14`, `pipelines-docs-1.15`, `pipelines-docs-1.16`

Issue: https://issues.redhat.com/browse/RHDEVDOCS-6146

Link to docs preview: https://82210--ocpdocs-pr.netlify.app/openshift-pipelines/latest/tkn_cli/op-tkn-reference.html

QE review:
- [ ] QE has approved this change.

Additional information:
Updated the document according to [RHDEVDOCS-6146](https://issues.redhat.com/browse/RHDEVDOCS-6146)

### CODE REVIEW

**Comment by Maharshi-Project** [comment]
@eromanova97 
Can you please review this PR ?

**Comment by Maharshi-Project** [comment]
@eromanova97 
Can you please review this PR ?

File: modules/op-tkn-condition-management.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 = Condition management commands
```
Comment (Srivaralakshmi) [inline modules/op-tkn-condition-management.adoc:5]: Good job! I have a suggestion about the content type attribute. Please implement the suggestion given.

After L2, leave a line. Before L4, add this attribute:
:_mod-docs-content-type: REFERENCE

For more information, see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes and https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata.

Thanks, @Maharshi-Project.

Comment (Maharshi-Project) [reply modules/op-tkn-condition-management.adoc:7]: I have updated the file. Could you please verify it?

Comment (Srivaralakshmi) [reply modules/op-tkn-condition-management.adoc:7]: LGTM

**Comment by Srivaralakshmi** [comment]
Hello @Maharshi-Project: I see 9 commits now. There seems to be some issue with rebasing (syncing) with the `pipelines-docs-main` branch, which caused multiple commits and changes appearing in your PR. 

It is difficult to help with this fix from distance, but I will try my best. You can try these steps to fix it:

````
$ git checkout <your_feature_branch> #(for example, git checkout feature-pipeline-docs-main)
$ git fetch upstream
$ git rebase upstream/pipelines-docs-main
$ git push -f origin <your_feature_branch> #(for example, git push -f origin feature-pipeline-docs-main)
````
If something goes wrong (you'll see some merge conflicts when you run the `git rebase upstream/pipelines-docs-main` command), then the easier option is to update your local repository and start your PR from scratch:
````
$ git checkout pipelines-docs-main
$ git fetch upstream
$ git rebase upstream/pipelines-docs-main
$ git push origin pipelines-docs-main
$ git checkout -b <new_feature_branch>
````
And make changes again on the new up-to-date feature branch :slightly_smiling_face: Thank you!

**Pull request closed without merging**